api/app/routers/hfi_calc.py

```diff
@@ -140,6 +303,7 @@ async def get_hfi_results(request: HFIResultRequest,
                           response: Response,
                           token=Depends(authentication_required)):
     """ Returns calculated HFI results for the supplied details in the POST body """
+    # THIS FUNCTION IS DEPRECATED.
```

Comment:
any reason for not deleting this function right now?

Reply:
It's technically not deprecated I guess - it's still in use, doesn't have a replacement but will soon.

Reply:
It will get deleted in a subsequent PR as part of the ticket I'm working on.